invenio_records_lom/resources/serializers/schemas.py

```diff
@@ -135,7 +135,9 @@ class LOMUIObjectSchema(Schema):
 
     location = Location(attribute="metadata.technical.location")
 
-    rights = Rights(attribute="metadata.rights")
+    rights = Rights(attribute="metadata.rights.description.langstring")
```

Comment:
What is the reason for this change?

Reply:
Looking at the code of `Rights` (which this PR does not change), `Rights` clearly needs a langstring as input (which `metadata.rights` isn't).
As I see it, the unchanged code couldn't really have worked (it sure didn't for me).

Reply:
The following shows the strucutre of `metadata.rights` (due to LOM-UIBK):
![image](https://user-images.githubusercontent.com/86989418/164422191-694f3111-2400-4a15-9d70-e267a0ca7dea.png)
Note that the CC-url is situated at `metadata.rights.description.langstring.#text`.

However, before this change `Rights` was passed `metadata.rights` which it then took the `.url`-attribute of - so `metadata.rights.url` in total.
As I see it, the unchanged code couldn't really have worked (at least it failed when I tested it).